modules/integration_aws-redshift/conf/readme.yaml

```diff
@@ -0,0 +1,5 @@
+documentations:
+  - name: AWS metrics for Redshift
+    url: 'https://docs.splunk.com/Observability/gdi/get-data-in/connect/aws/aws-metrics.html#amazon-redshift'
```

Comment:
perhaps you can add the official aws doc which can lead to improvements or new detectors in future ? https://docs.aws.amazon.com/redshift/latest/mgmt/metrics-listing.html